### PR TITLE
zh-CN: General improvements on translations

### PR DESCRIPTION
**Pull request by mrmagic2020**
Applying for issue(s):
- None

This is a general improvement for `zh-CN` translations under progress. Changes include improving translation accuracy and readability, correcting misspelled characters, conforming to language conventions, and other enhancements.

**Current Progress:** All done.

### CODE REVIEW

**Comment by mrmagic2020** [comment]
@Gymnasiast This pull request is ready for review. Thanks!

File: data/language/zh-CN.txt [hunk 1]
```diff
@@ -4137,21 +4137,14 @@ STR_DTLS    :
...
+STR_PARK    :时空堡垒
 STR_DTLS    :
 
 ###########
 # Scenery #
 ###########
 
-## Start OpenRCT2 Official
-
-[TTPIRF05]
```
Comment (Gymnasiast) [inline data/language/zh-CN.txt:4150]: Please keep this.

**Comment by mrmagic2020** [comment]
@Gymnasiast Done :)

**Review by Gymnasiast — APPROVED** [review]

**Comment by Gymnasiast** [comment]
Merged, thanks!

**Pull request merged**